contracts/mocks/SafeTracker/SafeTrackerMockPool.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.18;
+
+import "../common/PoolMock.sol";
+import "../../interfaces/IPriceFeedOracle.sol";
+
+contract SafeTrackerMockPool is PoolMock {
```

Comment:
Rename to `STMockPool.sol` please. It's a convention we're trying to adapt to but it's not entirely adopted